### PR TITLE
Extract strings from support power name and description

### PR DESCRIPTION
**Pull request by Mailaender**
Completes https://github.com/OpenRA/OpenRA/issues/10475

### CODE REVIEW

File: mods/cnc/languages/campaign/en.ftl [hunk 1]
```diff
@@ -11,6 +11,11 @@ options-difficulty =
 ## campaign-maprules.yaml
 actor-moneycrate-name = Money Crate
 
+actor-airstrike-proxy =
+   .name = Air Strike
+   .description = Deploy an aerial napalm strike.
+    Burns buildings and infantry along a line.
+
```
Comment (PunkPun) [inline mods/cnc/languages/campaign/en.ftl]: it seems this was added 3 times, in campaign, in regular rules, and in map twist of fate. As regular translations are present by default the others can be deleted

**Review by PunkPun — COMMENTED** [review]
Otherwise LGTM

File: mods/cnc/languages/campaign/en.ftl (hunk 1 above)
Comment (Mailaender) [reply mods/cnc/languages/campaign/en.ftl]: Dedupcliated.

**Review by PunkPun — APPROVED** [review]
LGTM

**Comment by PunkPun** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/a85ef806b1191258a1213047554604509fdc9b8c...8d95377e94af8510b262e9d7fa96430c74b24fc0)

**Pull request merged**